code/game/atoms.dm

```diff
@@ -50,7 +50,7 @@
 	if(use_preloader && (src.type == _preloader.target_path))//in case the instanciated atom is creating other atoms in New()
 		_preloader.load(src)
 	. = ..()
-	attempt_init(...)
+	attempt_init(arglist(args))

```

Comment:
Note: I made this change because the devil friend's arguments would not pass on to the Initialize function with the original code setup. Was the ellipsis here for a reason?